File: LeetCode/771 Jewel and Stones.java
class Solution {
    public int numJewelsInStones(String J, String S) {
        
        int count = 0;
        
        for (char ch : S.toCharArray()) {
            if (J.indexOf(ch) >= 0)
                count += 1;   
        }
        
        // for (int i = 0; i < S.length(); i++) {
        //     if (J.contains(Character.toString(S.charAt(i))))
        //         count++;
        // }
        
        return count;
        
    }
}

File: LeetCode/771 Jewel and Stones.js
/**
 * @param {string} J
 * @param {string} S
 * @return {number}
 */
var numJewelsInStones = function(J, S) {
    
	var count = 0;

	for (let i = 0; i < S.length; i++) {
		if (J.indexOf(S.charAt(i)) > -1)
			count += 1;
	}

	return count;

};